recipes/easylzma/all/test_package/test_package.c

```diff
@@ -25,12 +24,11 @@ int main(){
                                format, inLen);
 
     if (rc != ELZMA_E_OK) {
-        //std::cout << "Config failed with error code: " << rc << std::endl;
```

Comment:
Good catch. I forgot to delete it when I ported the test_package from cpp to c.

Reply:
Yeah, reviewing a pr is like scoring a test as a teacher: it's always easier to point at problems and removing lines :smile:

Reply:
hahaha so true!